### DISCUSSION TITLE
All Devs Squad discussion - Febraury 2022

### DISCUSSION THREAD

**Discussion by ankushdharkar** [Real-Dev-Squad/website-todo-items#53] [Ideas]
How to make product and engineering?

**Comment by rohan09-raj** [comment]
I am taking two updates for a task assigned at interval of 5 days (they can show the draft PR if they have done something) and further more 2 days and then reassigning to other people who want tasks if there is no response

**Reply by Priya2303** [reply]
Am interested

**Comment by ankushdharkar** [comment]
E2E Tests

**Reply by ankushdharkar** [reply]
@whyDontI To lead this initiative

**Comment by ankurnarkhede** [comment]
Using the "Github projects" (https://github.com/Real-Dev-Squad/website-backend/projects/1) for kanban till we get to status site? 🤔
It will be useful in tracking tasks. We tend to forget the ongoing tasks if not tracked anywhere. We can migrate to status site once its ready.
Opinions?

**Comment by harshith-venkatesh** [comment]
Could AOs also be allowed to create tasks in dashboard-site along with creating issues in GitHub and assigning it to devs

**Reply by ankushdharkar** [reply]
Yes, maybe we need a new task type.

**Comment by harshith-venkatesh** [comment]
Till we have the identity service, could we look into a way where we specify the newcomers probably set of actions to get them familiarized with basic tasks / creating PR / Code Reviews?

I heard few vocal comments where few are considering code reviews as something that would be done by AOs or admins

**Reply by harshith-venkatesh** [reply]
AOs if requested for code reviews should nominate new developers of RDS or others as initial reviewer and take the final review before merging the changes

**Comment by rohan09-raj** [comment]
We can prepare multiple docs for the discord server for new members because we keep repeating same thing to people and we have to explain everytime.
The doc will contain all the steps and giving a idea of what our server do and how to familiarise yourself with Discord server etc.

**Reply by harshith-venkatesh** [reply]
Could we add link to blogs / videos here too
https://github.com/Real-Dev-Squad/website-todo-items/discussions/53#discussioncomment-2117333

**Comment by ankushdharkar** [comment]
Monthly upgrades to packages

**Comment by harshith-venkatesh** [comment]
Could we have Design documentation/ Repo where we could specify more about having a consistent color palette, a set of images that could be utilized everywhere, a source of truth on design for RDS?

**Reply by harshith-venkatesh** [reply]
https://github.com/Real-Dev-Squad/website-todo-items/discussions/53#discussioncomment-2117333

**Comment by ankushdharkar** [comment]
Channel for documentation

**Reply by harshith-venkatesh** [reply]
Github: https://github.com/Real-Dev-Squad/docs
Discord: https://discord.com/channels/673083527624916993/939575674010746990

**Comment by ankushdharkar** [comment]
Monthly AO sync up
